Mathlib/Analysis/RCLike/Inner.lean

```diff
@@ -124,11 +124,12 @@ lemma wInner_const_right (f : ι → 𝕜) (a : 𝕜) :
     ⟪f, const _ a⟫ₙ_[𝕜] = a * (𝔼 i, conj (f i)) := by simp [wInner_cWeight_eq_expect, mul_expect]
 
 lemma wInner_one_eq_inner (f g : ι → 𝕜) :
-    ⟪f, g⟫_[𝕜, 1] = ⟪WithLp.toLp 2 f, WithLp.toLp 2 g⟫_𝕜 := by
-  simp [wInner]
+    ⟪f, g⟫_[𝕜, 1] = ⟪(WithLp.equiv 2 _).symm f, (WithLp.equiv 2 _).symm g⟫_𝕜 := by
```

Comment:
Can you explain the reason for that change?

Reply:
This was a bad resolution of a merge conflict I think